frontend/src/customPropTypes/mines.js

```diff
@@ -1,15 +1,18 @@
 import { PropTypes, shape, arrayOf } from "prop-types";
 import { minePermit } from "@/customPropTypes/permits";
+import { mineExpectedDocument } from "@/customPropTypes/documents";
 
 // This file is anticipated to have multiple exports
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
It doesn't look like this comment is needed anymore either now that there are multiple exports. Could you remove this one two? Thanks